lib/text.rb

```diff
@@ -0,0 +1,17 @@
+class Text
+
+  def initialize
+
+    require 'twilio-ruby'
+
+    account_sid = '****************'
```

Comment:
Explore storing data as environment variables on your machine, and referencing them by their key within your app.
> https://www.rubyguides.com/2019/01/ruby-environment-variables/